pkg/api/playbook/playbook_api.go

```diff
@@ -3,22 +3,32 @@ package playbook
 import (
 	"io"
 	"net/http"
+	"reflect"
+	"soarca/internal/controller/database"
+	"soarca/internal/logger"
 	"strconv"
 
-	"soarca/internal/controller/database"
 	playbookrepository "soarca/internal/database/playbook"
 
 	"github.com/gin-gonic/gin"
 )
 
-// A PlaybookController implements the playbook API endpoints is dependent on a database.
-type playbookController struct {
+var log *logger.Log
+
+type Empty struct{}
+
+func init() {
+	log = logger.Logger(reflect.TypeOf(Empty{}).PkgPath(), logger.Info, "", logger.Json)
+}
+
+// a playbookhandler implements the playbook api endpoints is dependent on a database.
+type playbookHandler struct {
 	playbookRepo playbookrepository.IPlaybookRepository
 }
 
-// NewPlaybookController makes a new instance of playbookControler
-func NewPlaybookController(controller database.IController) *playbookController {
-	return &playbookController{playbookRepo: controller.GetDatabaseInstance()}
+// newplaybookcontroller makes a new instance of playbookcontroler
```

Comment:
Update description to match capitalisation of function. Maybe even be more descriptive.